sbin/submitty_autograding_shipper.py

```diff
@@ -234,8 +234,11 @@ def prepare_job(my_name,which_machine,which_untrusted,next_directory,next_to_gra
 
     # log completion of job preparation
     obj = packer_unpacker.load_queue_file_obj(JOB_ID,next_directory,next_to_grade)
-    partial_path = os.path.join(obj["gradeable"],obj["who"],str(obj["version"]))
-    item_name = os.path.join(obj["semester"],obj["course"],"submissions",partial_path)
+    if not obj["generate_output"]:
```

Comment:
Make all these if statements if defined (generate output) and generate_output == true 
[in python syntax of course]